app/jobs/mastodon_sync_list_job.rb

```diff
@@ -0,0 +1,50 @@
+class MastodonSyncListJob < ApplicationJob
```

Comment:
This ports over the logic from `mastodon_sync_list.rb` verbatim.

This was somewhat challenging to test locally, given that it cares about prod Mastodon keys &c. Might wanna kick the tires a tad.

Reply:
A recent project of mine has been to wire up Telebugs for better production error reporting. One benefit of your work to move all these into jobs is that now errors will flow into the tracker instead of into noisy cron emails that instantly get buried in my personal inbox.

Which I say to explain how I appreciate your contributions, and also to justify yoloing this into prod without testing because I agree that testing a prod integration is a pain.